File: jewels-and-stones.py
def numJewelsInStones(jewels, stones):
    count = 0
    for stone in stones:
        if jewels.find(stone) == -1:
            continue
        else:
            count += 1
    return count

jewels = "aA"
stones = "aAAbbbb"

print(numJewelsInStones(jewels, stones))

File: jewels-and-stones.js
const numJewelsInStones = (jewels, stones) => {
    stonesArr = stones.split('')
    let count = 0
    stonesArr.forEach(stone => {
        if (jewels.includes(stone)) {
            count ++
        }
    })
    return count
};

const jewels = "aA"
const stones = "aAAbbbb"

console.log(numJewelsInStones(jewels, stones))